src/sundials/sundials_adaptcontroller.c

```diff
@@ -66,7 +70,7 @@ SUNAdaptController SUNAdaptController_NewEmpty(SUNContext sunctx)
 
 void SUNAdaptController_DestroyEmpty(SUNAdaptController C)
```

Comment:
Should we update this to
```suggestion
void SUNAdaptController_DestroyEmpty(SUNAdaptController* C)
```

Reply:
Yes, but I think in a different PR.